tools/amd_build/build_amd.py

```
@@ -201,6 +201,68 @@ def remove_hcc(line: str) -> str:
                     sources.write(line)
             print(f"{hip_platform_file} updated")
 
+
+ck_kernel_launch_file = (
+    "third_party/composable_kernel/include/ck_tile/host/kernel_launch.hpp"
+)
+
+
+def add_make_kernel_pt(line: str) -> str:
+    search_string = "namespace ck_tile {"
+    replace_string = """namespace ck_tile {
+// Added by hipification to become a no-op on non supported architectures
+template <int MaxThreadPerBlock, int MinBlockPerCu, typename Kernel, typename... Args>
+#if CK_TILE_USE_LAUNCH_BOUNDS
+__launch_bounds__(MaxThreadPerBlock, MinBlockPerCu)
```

we need to throw an error rather than just silent passing.